Platform/ElkhartlakeBoardPkg/AcpiTables/Dsdt/MipiCamSensors.asl

```diff
@@ -510,20 +507,19 @@ Scope(\_SB.PC00) {
         If (LEqual (Arg2, 5)) { Return (GPPI(C2F3, Add(Multiply(32, C2G3), C2P3), C2I3, C2A3)) }
       }
       If(LEqual(PCHS, PCHL)) { // PCH-LP
-        Include ("IpuImgClkDsm.asl")
+        //Include ("IpuImgClkDsm.asl")
       }
       Return (Buffer() {0})
     }
   }
 
   Device (DSC3) { // Control Logic Device 3 when Discrete is selected
     Name(_ADR, Zero)
-    Name(_HID, "INT3472")
     Name(_CID, "INT3472")
     Name(_DDN, "PMIC-CRDG")
     Name(_UID, 3)
     Method(_CRS, 0x0, NotSerialized) {
-      Store (ResourceTemplate() {}, Local0)
+      Store (Zero, Local0)
```

Comment:
Is this fix correct ?  I guess _CRS will expect a package to return.  Why do we return 0 if none of the "if" condition is met.

Reply:
Submitted patch to https://github.com/slimbootloader/slimbootloader/pull/1432